ui/react-components/components/EditAppointment/EditAppointment.jsx

```diff
@@ -261,7 +263,7 @@ const EditAppointment = props => {
     };
 
     const setViewDateAndShowSuccessPopup = startDate => {
-        setViewDate(startDate.startOf('day').toDate());
+        startDate ? setViewDate(moment(startDate).startOf('day').toDate()) : setViewDate(moment().startOf('day').toDate());
```

Comment:
if the difference between the two cases is simply the date going into moment(), why not do that?

const date = startDate ? moment(startDate) : moment();
setViewDate(date.startOf('day').toDate())

This makes it clear what is being done

Reply:
updated